src/integration_tests/datasets/e2e_dataset_integration_test.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+from unittest import TestCase
+
+from config.config_factory import ConfigFactory
+
+from src.integration_tests.helpers.integration_helpers import (
+    MOCK_UNIT_RESPONSE,
+    cleanup,
+    create_dataset,
+    generate_headers,
+    load_json,
+    setup_session,
+)
+
+config = ConfigFactory.get_config()
+UNIT_ID = "43532"
```

Comment:
It might be better to put all the hard-coded data in testing in test data

Reply:
Will move